tidb-server/main.go

```diff
@@ -760,6 +761,7 @@ func setGlobalVars() {
 	} else {
 		executor.GlobalMemoryUsageTracker.SetBytesLimit(int64(cfg.Performance.ServerMemoryQuota))
 	}
+	executor.Start()
```

Comment:
Why do we put this in `setGlobalVars`?
Seems a bit confusing.

Reply:
I hope it is globally unique.

Reply:
Why not in main()?

Reply:
+1. My main points are:
1. Currently, we only set up global states, i.e. setting global variables in `setGlobalVars`. But `executor.Start()` starts a background goroutine, which is quite different.
2. If we call `executor.Start()` here and `executor.Stop()` in `main()`, they won't pair up and will make the code a bit confusing.

Reply:
Done